README.md

```diff
@@ -109,14 +109,9 @@ If you find that you need to run specs that interact with a remote service, you'
 
 Anything listed in the `secret_keys` entry of that file will automatically be masked by VCR in newly-recorded cassettes.
 
-## Database
```

Comment:
I re-ordered and re-organized the steps to ensure the custom seed data is added to Elasticsearch.